File: C/1061.c
#include <stdio.h>

int main() {
    int d, h, m, s;
    int timestamp[] = {0, 0};

    for (int i = 0; i < 2; i++) {
        int temp;
        scanf("%*s %d", &temp);
        timestamp[i] += temp * 86400;
        scanf("%d", &temp);
        timestamp[i] += temp * 3600;
        scanf("%*s %d", &temp);
        timestamp[i] += temp * 60;
        scanf("%*s %d", &temp);
        timestamp[i] += temp;
    }

    s = timestamp[1] - timestamp[0];

    printf("%d dia(s)\n", d = s / 86400);
    s -= d * 86400;
    printf("%d hora(s)\n", h = s / 3600);
    s -= h * 3600;
    printf("%d minuto(s)\n", m = s / 60);
    s -= m * 60;
    printf("%d segundo(s)\n", s);

    return 0;
}

File: Python/1061.py
dhms = (86400, 3600, 60, 1)
time = tuple([int(input()[4:]) * dhms[0] + sum([int(v) * dhms[i+1] for i, v in enumerate(input().split(":"))])] for k in range(2))
time = time[1][0] - time[0][0]
final_time = []
for t in range(len(dhms)):
    final_time.append(time // dhms[t])
    time -= (time // dhms[t]) * dhms[t]
result = ("dia(s)", "hora(s)", "minuto(s)", "segundo(s)")
[print(final_time[a], result[a]) for a in range(len(final_time))]